### PR TITLE
fix: Making the daemons keep up the status.

### PR DESCRIPTION
**Pull request by Blu-J**

### CODE REVIEW

File: container-runtime/src/Adapters/Systems/SystemForEmbassy/polyfillEffects.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export class PolyfillEffects implements oet.Effects {
       this.manifest.volumes,
     )
     const daemon = dockerProcedureContainer.then((dockerProcedureContainer) =>
-      daemons.runDaemon()(
+      daemons.runCommand()(
```
Comment (dr-bonez) [inline container-runtime/src/Adapters/Systems/SystemForEmbassy/polyfillEffects.ts:119]: Why this change

File: core/startos/src/service/service_effect_handler.rs [hunk 2]
```diff
@@ -1037,14 +1057,15 @@ impl ValueParserFactory for Status {
...
 }
 async fn set_main_status(context: EffectContext, params: SetMainStatus) -> Result<Value, Error> {
     dbg!(format!("Status for main will be is {params:?}"));
     let context = context.deref()?;
     match params.status {
-        Status::Running => context.started(),
-        Status::Stopped => context.stopped(),
+        SetMainStatusStatus::Running => context.started(),
+        SetMainStatusStatus::Stopped => context.stopped(),
+        SetMainStatusStatus::Starting => context.stopped(),
```
Comment (dr-bonez) [inline core/startos/src/service/service_effect_handler.rs:1068]: Why add a varient if just treating as stopped?

Comment (Blu-J) [reply core/startos/src/service/service_effect_handler.rs:1068]: To indicate a state, that it is trying to getting things started. Could be more usefull or not, but for now just not doing anything with this information until I talk with you.

File: container-runtime/src/Adapters/Systems/SystemForEmbassy/polyfillEffects.ts (hunk 1 above)
Comment (dr-bonez) [reply container-runtime/src/Adapters/Systems/SystemForEmbassy/polyfillEffects.ts:119]: ok, as long as this `runCommand` isn't collecting output, we don't want to be leaking memory buffering logs

**Review by dr-bonez — APPROVED** [review]

**Review by dr-bonez — APPROVED** [review]
